### PR TITLE
Some UI tweaks and route navigations fixed.

### PR DESCRIPTION
**Pull request by akj-333**
Some UI tweaks and route navigations fixed.

### CODE REVIEW

File: app/views/discounts/index.html.erb [hunk 1]
```diff
@@ -1,46 +1,32 @@
...
-                      <%= link_to 'Delete', discount_path(discount), method: :delete %>
-                    </div>   
-                  </td>   
-                </tr>   
-          <% end %>
-        </tbody>
-      </table>
+    <tbody>
+      <% @discounts.each do |discount| %>
+      <tr>
```
Comment (rafiu-accenture) [inline app/views/discounts/index.html.erb]: Please move this to partial

File: app/views/product_categories/index.html.erb [hunk 2]
```diff
@@ -1,45 +1,28 @@
...
 
-      <table class="table table-striped">
-        <thead>
-          <tr>   
-            <th>Name</th>    
-            <th>Actions</th>   
-          </tr>  
-        </thead>
+    <tbody>
+      <% @product_categories.each do |productcategory| %>
```
Comment (rafiu-accenture) [inline app/views/product_categories/index.html.erb]: Please move this to Partial

File: app/views/products/index.html.erb [hunk 3]
```diff
@@ -1,58 +1,41 @@
...
+        <th>Description</th>
+        <th>Product Category</th>
+        <th>Stock Status</th>
+        <th>Discount</th>
+        <th>Actions</th>
+        <th colspan="7"></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @products.each do |product| %>
```
Comment (rafiu-accenture) [inline app/views/products/index.html.erb]: convert to partial

File: app/views/stock_statuses/index.html.erb [hunk 4]
```diff
@@ -1,41 +1,26 @@
...
+  <h1>Stock Status</h1>
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>Status</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @stock_statuses.each do |stockstatus| %>
```
Comment (rafiu-accenture) [inline app/views/stock_statuses/index.html.erb]: move to partial

**Review by atreyee-99 — APPROVED** [review]

**Review by anjali13942 — APPROVED** [review]
Approved

File: app/views/discounts/index.html.erb (hunk 1 above)
Comment (akj-333) [reply app/views/discounts/index.html.erb]: Completed in the recent commit

File: app/views/product_categories/index.html.erb (hunk 2 above)
Comment (akj-333) [reply app/views/product_categories/index.html.erb]: Completed in the recent commit

File: app/views/stock_statuses/index.html.erb (hunk 4 above)
Comment (akj-333) [reply app/views/stock_statuses/index.html.erb]: Completed in the recent commit

File: app/views/products/index.html.erb (hunk 3 above)
Comment (akj-333) [reply app/views/products/index.html.erb]: Completed in the recent commit

**Pull request merged**